src/tex/ms.tex

```diff
@@ -226,14 +226,51 @@ \subsection{Symbolic regression}
 Loosely inspired by \textit{eureqa},
 \pysr searches the space of trees 
 containing a basis set of specified operators, variables, and arbitrary real constants
-to find an expression that best matches a given dataset.
+to find expressions which fit relations in a given dataset.
 \pysr uses ``regularized evolution'' for its search strategy 
 \cite{realRegularizedEvolutionImage2019,realAutoMLZeroEvolvingMachine2020},
 which itself is a variant on classic tournament selection
 \cite{brindleGeneticAlgorithmsFunction1980,goldbergComparativeAnalysisSelection1991}.
 This algorithm simulates natural selection: each expression is represented by a tree, corresponding to its ``genetic code.''
 Expressions are subsampled, and the fittest expression in each group (using a metric which combines accuracy and simplicity) is used to breed new mutated expressions, as well as cross-bred with other top expressions.
-\pysr adds several additional techniques onto these classic algorithms, such as explicit constant optimization using BFGS \cite{fletcherPracticalMethodsOptimization1988}, and an adaptive complexity penalty, while allowing the use of custom operators and losses which is important for this work.
+\pysr adds several additional techniques onto these classic algorithms, such as explicit constant optimization using the \textit{BFGS} optimization algorithm \cite{fletcherPracticalMethodsOptimization1988}, and an adaptive complexity penalty, while allowing the use of custom operators and losses which is important for this work.
+
+Internal to \pysr, a multi-objective optimization problem is solved: the evolution seeks to find expressions which are as accurate as possible, while being as simple as possible.
```

Comment:
I thought this part was already in the document... Did it get removed, or did the commits get overwritten or something?

Reply:
I didn't touch this part. Did you add it twice or moved it around?

Reply:
Weird. I'm not sure. Oh well, it should be all there!